app/models/jurisdiction.rb

```diff
@@ -15,7 +15,7 @@ class Jurisdiction < ApplicationRecord
 
   # All patients are all those in this or descendent jurisdictions
   def all_patients
-    Patient.includes([:jurisdiction]).where(jurisdiction_id: subtree_ids)
+    Patient.includes([:jurisdiction]).where(purged: false, jurisdiction_id: subtree_ids)
```

Comment:
This is an important change I want to highlight. As it is before this change, this includes purged records and is used by `viewable_patients` which is called almost everywhere for accessing the user's accessible patients which is problematic. We get around it by using the Patient scopes which explicitly check for purged, but this seems like a needed change in the case where purged may not have been checked and this is used. We just need to make sure this won't negatively affect our current indexes. @Bialogs Want to make you aware of this change as well.